cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -1008,8 +1019,8 @@ mod test {
             Err(YaccParserError {
                 kind: YaccParserErrorKind::IllegalString,
                 span,
-            }) => assert_eq!(span, Span::new(28, 28)),
-            Err(e) => panic!("Incorrect error returned {}", e),
+            }) if span == Span::new(28, 28) => (),
```

Comment:
Now that we have all the line/column helpers, instead of hard coding these spans we should probably check against those?

Reply:
Finally had some time to figure out the bug that i ran into implementing this...